### ISSUE TITLE
Support NFC Requires Authentication parameter

### ISSUE THREAD

**Issue by chris-maheu** [drallgood/jpasskit#271]
https://developer.apple.com/documentation/walletpasses/walletpassformat/nfc

Require additional authentication to release the NFC message
Passes don't require additional authentication to send the NFC payload to the NFC reader, but there may be scenarios where you would want to require additional authentication in the form of an incremental Face ID, Touch ID, or fallback passcode. An example would be on a high value loyalty card or ticket to reduce friendly fraud.

Developers can use the 'requiresAuthentication' key in the pass JSON to require additional authentication from users before presenting a pass to the NFC reader:

- 'requiresAuthentication' is an NFC dictionary key
- Only values of 'true' and 'false' are supported
- The default value is 'false' if the key is omitted
- Used to determine whether or not additional authentication is required for an NFC pass
- This feature increases friction for users trying to present their pass to an NFC reader so developer should be cognizant of the impact to the customer experience.

Example:

```
"nfc" : {
  "nnessage":"ACCOUNTNUMBER12345",
  "encryptionPublicKey":"MDkwEwYHKoZlzjOCAQYIKoZlzjODAQcDIgADFYg226s2NaVSH kb/OubW9Wn0f5ZJnnYzIGgqxfJR11ec=", 
  "requiresAuthentication":"true"
}
```

**Comment by drallgood** [comment]
Thanks @chris-maheu for reporting this. 
I’m pretty busy right now, but I’ll see when I can squeeze this in. 
Contributions are always welcome of course 😊

**Comment by drallgood** [comment]
Closed via #276

**Comment by drallgood** [comment]
Thanks @chris-maheu for the contribution. 
And sorry for the delay!
